pylibs/plotting/Histogram.py

```diff
@@ -54,10 +55,22 @@ def setup(self, sample):
     self.hist.Rebin(self.rebin)
     self.hist.Scale(1./self.rebin)
 
+  def setupRatio(self, sample):
+    if sample.type == SampleType.background:  
+      color = sample.fill_color
+    else:
+      color = sample.line_color
+    self.hist.SetLineColor(color)
+    self.hist.SetMarkerColor(color)
+
 @dataclass
 class Histogram2D:
   name: str = ""
   title: str = ""
+  log_x: bool = False
+  log_y: bool = False
+  log_z: bool = False
+  norm_type: str = "norm1"
```

Comment:
Can we use the standardized normalization types from `HistogramNormalizer` here?

Reply:
I am using the standard to_lumi normalisation for my 2D plots now and it's running. I haven't tried all other normalization types yet, but I think normalizations like to 1 has to be looked over also for 1D hists as well

Reply:
Ok, so how about we just use to_lumi, to_one, etc., here instead of strings, and perhaps print a message in case the implementation of normalization for a given histogram type is missing or not working properly? And then we worry about fixing HistogramNormalizer.